src/logic.js

```diff
@@ -12,7 +12,13 @@ define([
             "dataParent",
             "repeat_count",
             "filter",
-            "defaultValue"
+            "defaultValue",
+        ],
+        LABEL_REFERENCES = [
+            "helpItext",
+            "constraintMsgItext",
+            "labelItext",
+            "hintItext",
```

Comment:
Yeah. I thought about that for a bit since this is another thing that links javaRosa to core. Decided to punt on it since it would be more effort to try to route these through the plugin system for now.